utils/data.py

```
@@ -0,0 +1,73 @@
+def save_json(filename: str, data: List[Dict[str, Any]]) -> None:
+    """
+    리스트 데이터를 JSON 파일로 저장.
+    디렉토리가 없으면 자동으로 생성함.
+    """
+    file_path = DATA_DIR / filename
+
+    try:
+        file_path.parent.mkdir(parents=True, exist_ok=True)
+        with open(file_path, "w", encoding="utf-8") as f:
+            json.dump(data, f, ensure_ascii=False, indent=2)
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)